### ISSUE TITLE
Include the un-minified built files for easy debugging...

### ISSUE THREAD

**Issue by rubikzube** [shutterstock/rickshaw#6]
**Building from source was a snap, but not everyone wants to install node.  Also, I'm on v0.6.5 and had to run make twice:**

➜  rickshaw git:(master) make 
npm install clean-css
clean-css@0.3.0 ../node_modules/clean-css 
└── optimist@0.1.9
cat src/css/detail.css src/css/graph.css src/css/legend.css  > rickshaw.css
/Users/Rubikzube/node_modules/.bin/cleancss rickshaw.css > rickshaw.min.css

node.js:284
        throw new Error('process.stdout cannot be closed');
              ^
Error: process.stdout cannot be closed
    at SyncWriteStream.<anonymous> (node.js:284:15)
    at output (/Users/Rubikzube/node_modules/clean-css/bin/cleancss:43:7)
    at /Users/Rubikzube/node_modules/clean-css/bin/cleancss:24:5
    at [object Object].<anonymous> (fs.js:115:5)
    at [object Object].emit (events.js:64:17)
    at afterRead (fs.js:1111:12)
    at Object.wrapper [as oncomplete](fs.js:254:17)
make: **\* [rickshaw.min.css] Error 1

➜  rickshaw git:(master) make
npm install uglify-js
uglify-js@1.2.0 ../node_modules/uglify-js 
cat src/js/Rickshaw.js src/js/Rickshaw.Graph.js src/js/Rickshaw.Fixtures.Color.js src/js/Rickshaw.Fixtures.RandomData.js src/js/Rickshaw.Fixtures.Time.js src/js/Rickshaw.Fixtures.Number.js src/js/Rickshaw.Color.Palette.js src/js/Rickshaw.Graph.Ajax.js src/js/Rickshaw.Graph.Annotate.js src/js/Rickshaw.Graph.Axis.Time.js src/js/Rickshaw.Graph.Axis.Y.js src/js/Rickshaw.Graph.Behavior.Series.Highlight.js src/js/Rickshaw.Graph.Behavior.Series.Order.js src/js/Rickshaw.Graph.Behavior.Series.Toggle.js src/js/Rickshaw.Graph.HoverDetail.js src/js/Rickshaw.Graph.JSONP.js src/js/Rickshaw.Graph.Legend.js src/js/Rickshaw.Graph.RangeSlider.js src/js/Rickshaw.Graph.Renderer.Line.js src/js/Rickshaw.Graph.Renderer.Stack.js src/js/Rickshaw.Graph.Renderer.Bar.js src/js/Rickshaw.Graph.Smoother.js src/js/Rickshaw.Graph.Unstacker.js src/js/Rickshaw.Series.js  > rickshaw.js
/Users/Rubikzube/node_modules/.bin/uglifyjs rickshaw.js > rickshaw.min.js
